### DISCUSSION TITLE
VerneMQ HTTP API Get session information for a specific client_id

### DISCUSSION THREAD

**Discussion by iam-nagarajan** [vernemq/vernemq#1960] [Q&A] [answered]
Hi all,

I am looking for a way to get session information for a specific client ID using the HTTP APIs exposed.
There is an API https://docs.vernemq.com/live-administration/http-administration#retrieve-session-information to retrieve session information, but it lists all the sessions. 
The CLI equivalent has an option to filter the sessions by client_id like this
`vmq-admin session show --client_id test-client-1`

I tried sending client_id as the query parameter in the retrieve sessions API, but it didn't work.

**Comment by maartenl945** [comment, marked as answer]
What worked for me is: /api/v1/session/show?--client_id=test-client-1
